### PR TITLE
feat: use data from market+scenes+fixes

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

**Comment by eordano** [comment]
@AjimenezDCL please take this issue from here! I simplified it down to 1 method call, specifying the name of the scene, the tile type, and which parcels does the scene span.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadata.cs [hunk 1]
```diff
@@ -134,4 +84,34 @@ public static MinimapMetadata GetMetadata()
...
+        { MinimapMetadata.TileType.Loading, "#110e13" }
+    };
+
+    public static bool TryGetTileColor(MinimapMetadata.TileType type, out Color color)
+    {
+        color = Color.black;
+        if(!TileColorMapping.ContainsKey(type))
+            return false;
+
+        return !ColorUtility.TryParseHtmlString(TileColorMapping[type], out color);
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadata.cs]: We should probably cache the parsed colors.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs [hunk 2]
```diff
@@ -33,6 +33,14 @@ private void OnPlayerCoordsChange(Vector2Int current, Vector2Int previous)
...
     }
 
+    public void ProcessNewParcelData(Vector2Int coordinate, string newSceneName, MinimapMetadata.TileType type)
+    {
+        if (VERBOSE)
+        {
+            Debug.Log($"Updating {coordinate.x},{coordinate.y} to {newSceneName} (tyle type: {type})");
+        }
+    }
+
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs]: Is this ever called? I can't find any references

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections;
+using System.Collections.Generic;
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs]: These are not necessary, according to Rider.

**Comment by AjimenezDCL** [comment]
Dammit... Being @eordano the author of the PR I cannot reply to the conversations. Gonna tackle down the issues you are pointing, @javiermaria 

- Agree with the color caching.
- Indeed that method is not being called. Is a stub @eordano left for me and I forgot to remove it.
- Gonna remove unused using.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    public void UpdateMinimapSceneInformation(string scenesInfoJson)
+    {
+        var scenesInfo = Utils.ParseJsonArray<MinimapSceneInfo[]>(scenesInfoJson);
+        foreach (var scene in scenesInfo)
+        {
+            foreach (var parcel in scene.parcels)
+            {
+                MinimapMetadata.GetMetadata().SetTile(parcel.x, parcel.y, new MinimapMetadata.Tile(scene.type, scene.name));
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs]: Can we also cache this .GetMetadata or add `private MinimapMetadata minimapMetadata => MinimapMetadata.GetMetadata();` to the class?

**Review by javiermaria — APPROVED** [review]
I have tested it and it works as expected. The Unity code looks good

**Review by pravusjif — APPROVED** [review]
Tested the exposed build and everything worked as expected

**Pull request merged**